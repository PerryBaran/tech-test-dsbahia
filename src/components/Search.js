import React, { useState } from "react";
import "../styles/search.css";

const Search = () => {
  const [query, setQuery] = useState("");

  return (
    <div>
      <input
        type="text"
        className="search"
        placeholder="Enter keywords..."
        value={query}
        onChange={(e) => setQuery(e.target.value)}
      />
    </div>
  );
};

export default Search;
