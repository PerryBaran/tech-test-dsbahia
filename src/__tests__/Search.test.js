import React from "react";
import { render } from "@testing-library/react";
import Search from "../components/Search";

describe("Search", () => {
    it("Renders Correctly", () => {
        const { asFragment } = render(<Search />);
        expect(asFragment()).toMatchSnapshot();
    });
});