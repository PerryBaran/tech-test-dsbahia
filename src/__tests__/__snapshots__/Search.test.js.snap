// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Renders Correctly 1`] = `
<DocumentFragment>
  <div>
    <input
      class="search"
      placeholder="Enter keywords..."
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
